packages/core/lottie-react-native.podspec

```diff
@@ -2,6 +2,11 @@ require 'json'
 
 package = JSON.parse(File.read(File.join(__dir__, "package.json")))
 
+# Read the React Native version from the package.json relative to the podspec file
+react_package_file = File.expand_path("../react-native/package.json", File.dirname(__FILE__))
```

Comment:
I wonder if this would work for monorepos

Reply:
I know. Trying to wrap my head around it. It may not work if the directory is symlinked.

Reply:
Let's default it to true for now. If someone complains for monorepo, we will come back.